Mail/Views/New Message/Attachments/AttachmentsManager.swift

```diff
@@ -246,7 +246,7 @@ final class AttachmentsManager: ObservableObject {
         let attachmentsSlice = attachments[safe: 0 ..< draft.availableAttachmentsSlots]
 
         Task {
-            try? await self.parallelTaskMapper.map(collection: attachmentsSlice) { attachment in
+            await attachmentsSlice.concurrentMap { attachment in
                 _ = await self.importAttachment(attachment: attachment, disposition: disposition)
```

Comment:
Since this is @mainactor now I need to double-check the `importAttachment` func that now runs in the main thread.

Reply:
Reworked an split async code in `AttachmentsManagerWorker.swift` that I also moved to the `mail-core` package.